app/routes/preview.py

```diff
@@ -0,0 +1,88 @@
+import json
+import os
+
+import boto3
+from fastapi import APIRouter, Request
+from fastapi.responses import HTMLResponse
+from fastapi.templating import Jinja2Templates
+
+from ..crud.sync_db.tile_cache_assets import get_dataset_tile_caches
+from ..settings.globals import GLOBALS
+
+router = APIRouter()
+
+template_dir = os.path.join(
+    os.path.dirname(os.path.abspath(__file__)), os.pardir, "templates"
+)
+templates = Jinja2Templates(directory=template_dir)
+
+
+@router.get(
+    "/{dataset}/{version}/{implementation}/preview", response_class=HTMLResponse
+)
+async def get_tile_caches(
```

Comment:
Should this function be called something like "get_tile_cache_preview", just to clarify what it's returning? Since there's another function called get _dataset_tile_caches that just returns the metadata.